pkg/api/api.go

```diff
@@ -213,7 +215,7 @@ type IssueCodeRequest struct {
 
 // IssueCodeResponse defines the response type for IssueCodeRequest.
 type IssueCodeResponse struct {
-	Padding Padding `json:"padding"`
+	Padding Padding `json:"padding,omitempty"`
```

Comment:
this is a potentially breaking change - shouldn't be since we never leave blank on single issue, but perhaps worth noting.

Reply:
added to the pr description, but yes the single-issue should be unchanged